src/spring-cloud/azext_spring_cloud/application_configuration_service.py

```diff
@@ -0,0 +1,223 @@
+
+    logger.warning("[2/2] Adding item to Application Configuration Service settings, (this operation can take a while to complete)")
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, DEFAULT_NAME, acs_resource)
+
+
+def application_configuration_service_git_update(cmd, client, service, resource_group, name,
+                                                 patterns=None,
+                                                 uri=None,
+                                                 label=None,
+                                                 search_paths=None,
+                                                 username=None,
+                                                 password=None,
+                                                 host_key=None,
+                                                 host_key_algorithm=None,
+                                                 private_key=None,
+                                                 strict_host_key_checking=None,
+                                                 no_wait=False):
+    acs_resource = _get_or_default_acs_resource(client, resource_group, service)
+    repo = _get_existing_repo(acs_resource.properties.settings.git_property.repositories, name)
+    repo = _replace_repo_with_input(repo, patterns, uri, label, search_paths, username, password, host_key, host_key_algorithm, private_key, strict_host_key_checking)
+
+    _validate_acs_settings(client, resource_group, service, acs_resource.properties.settings)
+
+    logger.warning("[2/2] Updating item of Application Configuration Service settings, (this operation can take a while to complete)")
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, DEFAULT_NAME, acs_resource)
+
+
+def application_configuration_service_git_remove(cmd, client, service, resource_group, name, no_wait=False):
+    acs_resource = _get_or_default_acs_resource(client, resource_group, service)
+
+    repo = _get_existing_repo(acs_resource.properties.settings.git_property.repositories, name)
+    acs_resource.properties.settings.git_property.repositories.remove(repo)
+
+    _validate_acs_settings(client, resource_group, service, acs_resource.properties.settings)
+
+    logger.warning("[2/2] Removing item of Application Configuration Service settings, (this operation can take a while to complete)")
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, DEFAULT_NAME, acs_resource)
+
+
+def application_configuration_service_git_list(cmd, client, service, resource_group):
+    acs_resource = client.configuration_services.get(resource_group, service, DEFAULT_NAME)
+    acs_settings = acs_resource.properties.settings
+
+    if not acs_settings or not acs_settings.git_property or not acs_settings.git_property.repositories:
+        raise CLIError("Repos not found.")

```

Comment:
Please use the specific error type `ResourceNotFoundError` instead of `CLIError`.

Reply:
Updated, thanks for suggestion.